README.md

```diff
@@ -47,6 +47,8 @@ by InCommon that is authorized to create host certificates for that account.
 Usage: osg-incommon-cert-request [--debug] -u username -k pkey -c cert \
            (-H hostname | -F hostfile) [-a altnames] [-d write_directory]
        osg-incommon-cert-request [--debug] -u username -k pkey -c cert -t
+       osg-incommon-cert-request [--orgcode org,dept] (-H hostname | \
```

Comment:
I don't think this should be a third form, it's just optional parameters to the first form.  It's too long to fit the first form in 2 lines, but that's OK you can make it 3 lines.  Don't let it go beyond 80 characters per line.

You can use the short form -O here instead of --orgcode, that's what's done for the other options.  I think it will still be too long to fit everything on 2 lines, because the second line should stay indented beyond the beginning of the command name on the first line.

Reply:
Fixed.